src/libraries/System.Data.OleDb/src/PACKAGE.md

```diff
@@ -0,0 +1,47 @@
+## About
+
+This package implements a data provider for OLE DB data sources.
+
+## Key Features
+
+Allows access to legacy OLE DB data sources.
+
+## How to Use
+
+This is a basic example of retrieving the results of a query using an `OleDbDataReader`. For examples of using an `OleDbDataAdapter`, and of updating an OLE DB data source, please see the documentation.
+
+```cs
+string queryString = "SELECT OrderID, CustomerID FROM Orders";
+using (OleDbConnection connection = new OleDbConnection(connectionString))
+{
+    OleDbCommand command = new OleDbCommand(queryString, connection);
+    connection.Open();
+    OleDbDataReader reader = command.ExecuteReader();
+
+    while (reader.Read())
+    {
+        Console.WriteLine(reader.GetInt32(0) + ", " + reader.GetString(1));
+    }
+
+    reader.Close();
+}
```

Comment:
```suggestion
string queryString = "SELECT OrderID, CustomerID FROM Orders";

using OleDbConnection connection = new OleDbConnection(connectionString);
using OleDbCommand command = new OleDbCommand(queryString, connection);

connection.Open();
using OleDbDataReader reader = command.ExecuteReader();

while (reader.Read())
{
    Console.WriteLine(reader.GetInt32(0) + ", " + reader.GetString(1));
}
```

Reply:
ah I didn't do that as I didn't realize the base class Dispose() calls Close().